.gitignore

```diff
@@ -17,3 +17,4 @@ example/*
 !example/template/*.html
 npm-debug*
 package-lock.json
+.env
```

Comment:
this file is where I put env var... I use `source .env` before running commands.. you won't usually need this unless you are testing deployments.